### PR TITLE
Mars-ify

### PR DESCRIPTION
**Pull request by Azrun**
[feature]
## About the PR 
Cool RP tools for Cool kids.  PUNISHMENT TO THOSE ON CLASSIC FOR FAILING TO FOLLOW INSTRUCTIONS! 😈 
-Rescales _thematic_ elements to not **destroy** players immediately.
-Ambient light gradient used across station z-level to nievely approximate a sunrise, some randomization to avoid visible banding.

## Why's this needed? 
image

Can I? ✔️ 
Should I? 🤷

### CODE REVIEW

File: code/modules/admin/terrainify.dm [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 	var/options = list(
 		"Ice Moon Station"=/client/proc/cmd_ice_moon_station,
+    "Mars Station"=/client/proc/cmd_marsify_station,
```
Comment (Sovexe) [inline code/modules/admin/terrainify.dm:9]: woops
```suggestion
		"Mars Station"=/client/proc/cmd_marsify_station,
```

File: code/datums/mapgen/MarsGenerator.dm [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+		if (ishuman(O))
+			if(!ON_COOLDOWN(O,"mars_duststorm", rand(5 SECONDS, 15 SECONDS)))
+				var/mob/living/jerk = O
+				if (!isdead(jerk))
+					if((istype(jerk:wear_suit, /obj/item/clothing/suit/armor/mars))&&(istype(jerk:head, /obj/item/clothing/head/helmet/mars))) return
+					step(jerk,EAST)
+					if(jerk.protected_from_space()) // Be kind around station...
+						return
+					random_brute_damage(jerk, 20, checkarmor=TRUE) // Allow armor to resist
+					jerk.changeStatus("weakened", 5 SECONDS)
```
Comment (Sovexe) [inline code/datums/mapgen/MarsGenerator.dm:120]: possible alternative. If you don't like this then instead just be sure to add a `force_laydown_standup` after the weakened effect is addded 
```suggestion
					jerk.do_disorient(stamina_damage = 100, weakened = 3 SECONDS, disorient = 5 SECOND)
```

File: code/datums/mapgen/MarsGenerator.dm [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+					if(prob(50))
+						playsound(src.loc, 'sound/impact_sounds/Flesh_Stab_2.ogg', 50, 1)
```
Comment (Sovexe) [inline code/datums/mapgen/MarsGenerator.dm:122]:
```suggestion
						playsound(src, "sound/impact_sounds/Flesh_Stab_2.ogg", 50, 1)
```
Play the sound on the turf not the turf's area, also double quotes to use the sound cache

**Review by Sovexe — CHANGES_REQUESTED** [review]

**Comment by Azrun** [comment]
- [ ] Cleaning up handling the shipping path, as it appears possible for the `obj/machinery/light/beacon` and `/turf/unsimulated/wall/setpieces/martian/auto` to block.

**Review by Sovexe — APPROVED** [review]

**Pull request merged**